activerecord/test/cases/adapters/mysql2/active_schema_test.rb

```diff
@@ -180,6 +180,19 @@ def test_indexes_in_create
     assert_match expected, actual
   end
 
+  def test_forceful_temporary_table_creation
```

Comment:
Since this is not related to `mysql2` adapter directly, I think it would fit https://github.com/rails/rails/blob/main/activerecord/test/cases/migration/change_schema_test.rb test file better (including already various `create_table `tests).

Reply:
@simi Good point, thank you! I will add it there.

Reply:
I would like some help here.

- The target [test file](https://github.com/rails/rails/blob/main/activerecord/test/cases/migration/change_schema_test.rb) doesn't have checks for the produced SQL but only assertions on the newly created table whereas in my case I need to check that the `temporary` directive is being added to the query. Is it OK to introduce such checks there?

- I suppose I should add specs also for the rest of the adapters (currently I check only against mysql, sqlite & postgresql). Could someone point me on the documentation on how to setup the suite for the other adapters as well?